tstest/integration/testcontrol/testcontrol.go

```diff
@@ -454,6 +465,9 @@ func (s *Server) serveMap(w http.ResponseWriter, r *http.Request, mkey tailcfg.M
 		}
 		s.updates[nodeID] = updatesCh
 	} else {
+		if s.pingReqsToAdd[node.Key] != nil {
```

Comment:
not here. delete these three lines. it's the place above that sends this.

Reply:
does this mean in if-block instead of in the else-block, or just removed completely?